hircluster.c

```diff
@@ -3096,7 +3071,7 @@ void *redisClusterCommandToNode(redisClusterContext *cc, cluster_node *node,
     redisContext *c;
     va_list ap;
     int ret;
-    redisReply *reply;
+    void *reply;
```

Comment:
Why void*? Why not the actual type?

Reply:
The reply is now got from hiredis by `int redisGetReply(redisContext *c, void **reply)` which would need a cast like `(void**)&reply` to be able to give it as an argument.
Changed it to void* since this function returns as void* anyway.